src/index.ts

```diff
@@ -21,6 +21,8 @@ const app = Fastify({
   }
 })
 
+app.server.keepAliveTimeout = 5 * 60 * 1000 // 5 minutes
```

Comment:
why do we want to set the keepAliveTimeout so high? we should be trying to get rid of KeepAlive for our use-case here shouldn't we?

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive & https://httpwg.org/specs/rfc9113.html#ConnectionSpecific